AdblockPlusSafari.xcodeproj/project.pbxproj

```diff
@@ -804,6 +845,36 @@
 			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-AdblockPlusSafariTests/Pods-AdblockPlusSafariTests-resources.sh\"\n";
 			showEnvVarsInLog = 0;
 		};
+		83C19D903DD1D9F927CBD253 /* Copy Pods Resources */ = {
```

Comment:
Huh? I might understand what you asking but I am afraid that I am not able to split this commit to two standalone working parts. I can move xcode project file to different commit, and append this commit to this pull request. In that case you would have to pull both commits to be able to build it properly.